advanced-client-install.rst

```diff
@@ -33,12 +33,12 @@ Windows will by default have firewall rules that prevent inbound connections. If
 
 ``netsh advfirewall firewall add rule name="Allow from <peer private addr>" dir=in action=allow protocol=ANY remoteip=<peer private addr>``
 
-If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server.
+If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server from Netmaker GUI.
```

Comment:
point out that relays are a pro feature and they'll need a self-hosted server to accomplish this.

Reply:
Pushed a commit with amendments.